config/src/keys.rs

```diff
@@ -1,96 +1,42 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use libra_crypto::{PrivateKey, Uniform, ValidKeyStringExt};
-use mirai_annotations::verify_unreachable;
-use serde::{de::DeserializeOwned, Deserialize, Deserializer, Serialize, Serializer};
-
-#[derive(Debug, PartialEq)]
-#[cfg_attr(any(test, feature = "fuzzing"), derive(Clone))]
-pub enum PrivateKeyContainer<T> {
-    Present(T),
-    Removed,
-    Absent,
```

Comment:
note that I'm greatly simplifying a KeyPair by removing the `PrivateKeyContainer` structure and replacing it with an `Option`. Rational:

* We never use the `Removed` part, and after thinking about it and I don't see any good use-case for it. This is after all just for configs.
* an `Option` does pretty much the same thing with its built-in `take()` method.

Reply:
Yes! And more importantly storing keys in configs is an anti-pattern for testing only. There is no point in applying security best practices for what should only ever be used in test cases.